src/promptflow-tools/tests/test_common.py

```diff
@@ -354,3 +355,93 @@ def test_normalize_connection_config_for_aoai_meid(self):
             "azure_ad_token_provider": aoai_meid_connection.get_token
         }
         assert normalized_config == expected_output
+
+    @pytest.mark.parametrize(
+        "value, escaped_dict, expected_dict, expected_val",
```

Comment:
Hard to understand the relationship between `escaped_dict` and `expected_dict`.

Actually, `expected_dict` is the real escaped_dict, while `escaped_dict` is just input. And it is weird that all tests escaped_dict is empty.

Reply:
updated